test/wolff_supervised_tests.erl

```diff
@@ -485,18 +464,18 @@ kafka_topic_cmd_base(Topic) ->
     " --zookeeper zookeeper:2181" ++
     " --topic '" ++ Topic ++ "'".
 
-assert_eq_optional_tail(Fun, ExpectedList) ->
-    ExpectedListMinusLast = lists:sublist(ExpectedList, length(ExpectedList) - 1),
-    case Fun() of
-        ExpectedList ->
-            ok;
-        ExpectedListMinusLast ->
-            %% sometimes the terminate callback seems to not have time
-            %% to insert into the ets, making this flaky.
-            ok;
-        Xs0 ->
-            ct:fail("unexpected result: ~p", [Xs0])
-    end.
-
 get_telemetry_seq(Table, Event) ->
     wolff_tests:get_telemetry_seq(Table, Event).
```

Comment:
Not related to this PR but you can fix it if you want. There seem to be a mix between calling wolff_tests:get_telemetry_seq/2 and the local function get_telemetry_seq/2. I think it would be better to consistently use the same so the reader don't need to think about if they are different or not.

Reply:
fixed.